bucket/eiskaltdcpp.json

```
@@ -0,0 +1,38 @@
+{
+    "version": "2.4.2",
+    "description": "File sharing program using DC and ADC protocols",
+    "homepage": "https://github.com/eiskaltdcpp/eiskaltdcpp",
+    "license": "GPL-3",
```

```suggestion
    "license": "GPL-3.0-or-later",
```
check https://spdx.org/licenses/ for license identifiers.

---

👍